src/index.js

```diff
@@ -1,5 +1,40 @@
 import $ from 'jquery'
 
 $(document).ready(() => {
-  // have fun!
+  fetch('https://wordwatch-api.herokuapp.com/api/v1/top_word')
+    .then(response => response.json())
+    .then(json => topWord(json))
+    .catch(error => console.log(error))
+
+  function topWord(data) {
+    var word = Object.keys(data.word)[0]
+    var value = data.word[word];
+    $('.top-word').append(`<p>${word} is the most popular word </p>`)
+    $('.word-count').append(`<p>Count : ${value} </p>`)
+  }
+
+  $('#submit-word').on('click', function() {
```

Comment:
This function get super lengthy - I would recommend you pull out 21-35 into it's own helper for the request😊